api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-18.xml

```diff
@@ -2334,46 +2357,48 @@
-  <concept_set concept_set_id="1639" concept_id="162358" concept_set="162384" sort_weight="8.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1639AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1640" concept_id="162359" concept_set="162384" sort_weight="9.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1640AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1641" concept_id="161554" concept_set="162384" sort_weight="10.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1641AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1642" concept_id="162360" concept_set="162384" sort_weight="11.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1642AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1643" concept_id="162361" concept_set="162384" sort_weight="12.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1643AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1644" concept_id="162264" concept_set="162384" sort_weight="13.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1644AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1645" concept_id="162362" concept_set="162384" sort_weight="14.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1645AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1646" concept_id="162262" concept_set="162384" sort_weight="15.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1646AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1647" concept_id="162363" concept_set="162384" sort_weight="16.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1647AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1648" concept_id="162364" concept_set="162384" sort_weight="17.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1648AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1649" concept_id="162365" concept_set="162384" sort_weight="18.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1649AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1650" concept_id="162366" concept_set="162384" sort_weight="19.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1650AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1651" concept_id="161553" concept_set="162384" sort_weight="20.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1651AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1652" concept_id="162263" concept_set="162384" sort_weight="21.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1652AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
-  <concept_set concept_set_id="1653" concept_id="162367" concept_set="162384" sort_weight="22.0" creator="1" date_created="2014-05-15 20:36:58.0" uuid="1653AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
```

Comment:
Why did the sort_weights change?

Reply:
I'm not quite sure. This is what was generated by the data exchange module. Infact, all the changes so far were what was got as output